AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/Features.swift

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCLIUtils
+
+struct Features: Decodable {
+    let features: [Feature]
+
+    static func fromFile(_ filePath: String) throws -> Self {
+        var fileContents: Data
+        do {
+            fileContents = try FileManager.default.loadContents(atPath: filePath)
+        } catch {
+            throw Error("Failed to load build-request.json.")
+        }
+        do {
+            return try JSONDecoder().decode(Self.self, from: fileContents)
+        } catch {
+            throw Error("Failed to decode build-request.json.")
```

Comment:
Instead of throwing this error which hides the underlying problem, why not just let
```swift
try JSONDecoder().decode(Self.self, from: fileContents)
```
throw its own error?

Reply:
Good point; checked the original error thrown does print exact issue. Will do this for other catch clauses as well.